docs/topics/api/accounts.rst

```diff
@@ -187,6 +187,46 @@ This endpoint lists the add-ons in a collection, together with collector's notes
     :>json int results[].downloads: The downloads that occured via this collection.
 
 
+------------------
+Notifications List
+------------------
+
+.. _notification-list:
+
+.. note::
+    This API requires :doc:`authentication <auth>` and `Users:Edit`
+    permission to list notifications on accounts other than your own.
+
+This endpoint allows you to list the account notifications set for the specified user.
+The result is an unpaginated list of the fields below.  There are currently 11 notifications.
+
+.. http:get:: /api/v3/accounts/account/(int:user_id|string:username)/notifications/
+
+    :>json string name: notification short name
+    :>json boolean enabled: if the notification is enabled (defaults to True)
+    :>json boolean mandatory: if the notification can be set by the user.
+
+
+-----------------
+Notifications Set
+-----------------
+
+.. _`notification-set`:
+
+.. note::
+    This API requires :doc:`authentication <auth>` and `Users:Edit`
+    permission to set notification preferences on accounts other than your own.
+
+This endpoint allows account notifications to be set. The request should be a dict of `name`:True|False pairs.
+Any number of notifications can be set; only non-mandatory notifications can be changed - attempting to set a mandatory notification will be ignored.
```

Comment:
> attempting to set a mandatory notification will be ignored.

This is going make someone lose a lot of hair one day :) Can it return a [409](https://httpstatuses.com/409) instead of ignoring a request to change a mandatory notification?